ios/FluentUI/Bottom Commanding/CommandingItem.swift

```diff
@@ -53,6 +53,15 @@ open class CommandingItem: NSObject {
         }
     }
 
+    /// A `UIView` to be displayed on the trailing end.
```

Comment:
Would be nice to see this used in the objc demo as well, somehow, given it is new objc exposure.

Reply:
Offline Harrie mentioned adding to the BottomCommandingController demo page only so resolving for now but could be added going forward!